web/template/mlpa-payments.gohtml

```diff
@@ -8,9 +8,18 @@
 
             {{ template "mlpa-header" (caseTabs .Lpa "fees") }}
 
+            {{ if .FlashMessage.Title }}
+                {{ template "notification-banner" .FlashMessage }}
+            {{ end }}
+
             <h1 class="govuk-heading-l govuk-!-margin-bottom-5">Fees</h1>
 
-            <h2 class="govuk-heading-m govuk-!-margin-bottom-3">Summary</h2>
+            <div class="app-two-element-row">
+                <h2 class="govuk-heading-m govuk-!-margin-bottom-3">Summary</h2>
+                <a role="button" class="govuk-button govuk-button--secondary" data-module="govuk-button" href="{{ prefix (printf "/add-payment?id=%d" .Case.ID) }}">
+                    Add a payment
+                </a>
+            </div>
```

Comment:
Gemma [has something similar on her PR](https://github.com/ministryofjustice/opg-sirius-lpa-frontend/pull/376/files#diff-cbc819515725491bf5563f714aa33d322923d7c418f80eba3f30153c1e4e70c7R11-R25), it might be worth coordinating so these are consistent (though could be done after these PRs to avoid blocking each other)

Reply:
Right, I might just wait for @GemTay to merge that, then rework this PR to match.

Reply:
In fact, I can just modify what I've got to match Gemma's.